rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import babel from 'rollup-plugin-babel';
+
+var config = {
+  entry: 'src/index.js',
+  dest: 'dist/ReactOnClickOutside.js',
+  format: 'umd',
+  moduleName: 'onClickOutside',
+  exports: 'named',
+  external: ['react', 'react-dom'],
+  globals: {
+    react: 'React',
+    'react-dom': 'ReactDOM'
+  },
+  plugins: [
+    nodeResolve({
+      jsnext: true
+    }),
+    babel({
+      exclude: 'node_modules/**',
+    })
+  ]
+};
+
+export default config;
```

Comment:
Please update rollup (and other rollup-related dependencies) to its latest version. Rollup's tree-shaking algorithm has changed a little bit and for now I advise to filter out `asyncGenerator` helper like [here](https://github.com/redux-saga/redux-saga/blob/9382f74c3e490fb952ae6f8fe71e52fbc5609f97/rollup.config.js#L21)

The linked config was recently upgraded, you can find the commit and see what's needed by the new rollup API